app/src/main/java/com/github/dfa/diaspora_android/activity/MainActivity.java

```diff
@@ -1052,4 +1082,54 @@ public void onRequestPermissionsResult(int requestCode, @NonNull String[] permis
                         grantResults);
         }
     }
+
+    /**
+     * Ask the user whether to install Orbot or not. Check if installing from
+     * F-Droid or Google Play, otherwise take the user to the Orbot download
+     * page on f-droid.org.
+     */
+    void promptInstallOrbot() {
+        String message = this.getString(R.string.you_must_have_orbot) + "  ";
```

Comment:
final Intent intent = OrbotHelper.getOrbotInstallIntent(MainActivity.this);
String message = getString(R.string.you_must_have_orbot) + "  " +
message += getString(intent.getPackage() == null ?
                       R.string.download_orbot_from_fdroid : R.string.get_orbot_from_fdroid);